File: java/chaincue-real-estate/src/main/java/com/chaincuerealestate/chaincuerealestate/repositories/HouseRepository.java
package com.chaincuerealestate.chaincuerealestate.repositories;

import com.chaincuerealestate.chaincuerealestate.domains.House;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface HouseRepository extends JpaRepository<House, String> {
}

File: kotlin/chaincue-real-estate-reactive/src/main/kotlin/com/chaincuerealestate/chaincuerealestatereactive/repositories/HouseRepository.kt
package com.chaincuerealestate.chaincuerealestatereactive.repositories
import com.chaincuerealestate.chaincuerealestatereactive.domains.House
import org.springframework.data.repository.kotlin.CoroutineCrudRepository
import org.springframework.stereotype.Repository

@Repository
interface HouseRepository : CoroutineCrudRepository<House, String>

